docs/getting-started/threat-intel-integrations.asciidoc

```diff
@@ -0,0 +1,50 @@
+
+[float]
+[[agent-ti-integration]]
+== Enable an {agent} integration
+
+To set up an {agent} integration:
+
+. Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet} managed {agent}] on the servers you want to monitor.
+. In the Threat Intelligence view, click *Enable sources*.
+. Select an integration, and complete the installation steps.
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+
+[float]
+[[ti-mod-integration]]
+== Enable a Threat Intel module integration
+
+To set up an integration through the Threat Intel module:
+
+. Set up the {filebeat-ref}/filebeat-installation-configuration.html[{filebeat} agent] and enable the Threat Intel module.
++
+NOTE: For more information about enabling available threat intelligence filesets, refer to {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module].
+
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`):
+.. If you're using {filebeat} version 8.x _only_, add the appropriate {filebeat} threat intelligence index pattern, for example, `filebeat-8*` or `filebeat*`.
+.. If you're using a previous version of Filebeat _and_ a current one, differentiate between the threat intelligence indices by using unique index pattern names. For example, if you’re using {filebeat} version 7.0.0 and 8.0.0, add `filebeat-7*` and `filebeat-8*`.
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+
+[float]
+[[custom-ti-integration]]
+== Enable a custom integration
+
+To enable a custom integration:
+
+. Set up a way to <<ingest-data, ingest data>> into your system.
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`), for example, `custom-ti-index*`.
++
+NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like for your alerts to be enriched with relevant threat indicator information. You can find a list of ECS compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
+
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
++
+NOTE: The Threat Intelligence view looks for a `threat.feed.name` field value to define the source name in the *Name* column. If a custom source doesn't have the `threat.feed.name` field, or hasn't defined a `threat.feed.name` field value, it's named *Other*. In addition, dashboards links are not typically not created for sources in the *Other* category unless a source has the `threat.feed.dashboard_id` field defined.
```

Comment:
Also, "not typically created" seems kind of vague -- any way we can explain more precisely?
```suggestion
NOTE: The Threat Intelligence view searches for a `threat.feed.name` field value to define the source name in the *Name* column. If a custom source doesn't have the `threat.feed.name` field or hasn't defined a `threat.feed.name` field value, it's named *Other*. In addition, dashboards links are not typically created for sources in the *Other* category unless a source has the `threat.feed.dashboard_id` field defined.
```